### DISCUSSION TITLE
my application rejected again

### DISCUSSION THREAD

**Discussion by assafkeller** [community/community#53112] [GitHub Education]
Hi 
i tried to register to github education, but im unable to verify your academic affiliation.
is it possible  to sent it by mail?
Thank you

**Comment by kottinov** [comment]
Hey, for Github educational pack you should be able to verify your student status via your university/college email. If that doesn't work for you, make sure to try get help through [Github support](https://support.github.com/)

**Reply by samus-aran** [reply]
Hi @kottinov 👋 Awesome to try and help out others in the community. I would only like to add we don't provide support for GitHub Global Campus manual reviews as [detailed here](https://github.com/orgs/community/discussions/49366) so we would suggest asking for more information from the original poster 👍
